CommandListener.cpp

```diff
@@ -67,7 +67,7 @@ void CommandListener::dumpArgs(int argc, char **argv, int argObscure) {
         if (i == argObscure) {
             len += 2; // Account for {}
         }
-        if (((p - buffer) + len) < (sizeof(buffer)-1)) {
+        if (((unsigned int)(p - buffer) + len) < (unsigned int)(sizeof(buffer)-1)) {
```

Comment:
Why is this needed? `p` never goes backwards unless someone intentionally passes a huge argument causing it to overflow. Also the patch isn't in CM upstream, maybe you could submit it to CM Gerrit if it's truly necessary?